maven-failsafe-plugin/pom.xml

```diff
@@ -127,6 +127,19 @@
               <excludeTransitive>true</excludeTransitive>
             </configuration>
           </execution>
+          <execution>
+            <id>shared-logging-generated-sources</id>
```

Comment:
Good point. I used m-shared-u:3.1.0 but there was compilation error I know from branch 3.0-rc1.
One class of m-shared-u appeared in another package related to report. I don't remember the name of the class but it can be easily reproduced.
This code would have very short life time because we already have branch `3.0-rc1` and one IT should be fixed and Surefire 2.x is gone.